synapse/lib/cell.py

```diff
@@ -575,6 +577,13 @@ async def _initNexsRoot(self):
         nexsroot.onfini(self)
         return nexsroot
 
+    async def _leaderHookGate(self):
```

Comment:
I still don't understand why you have two methods here. Just make 1. You're already taking the method call hit.